WireAPI/Sources/WireAPI/APIs/AccountsAPI/AccountsAPIV7.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class AccountsAPIV7: AccountsAPIV6 {
+    override var apiVersion: APIVersion {
+        .v7
+    }
+
+    override func upgradeToTeam(teamName: String) async throws -> UpgradedAccountTeam {
+        let components = URLComponents(string: "upgrade-personal-to-team")
+
+        guard let url = components?.url else {
```

Comment:
Optional: Personally I might force unwrap this and ensure it is tested

Reply:
Definitely an option, but I would rather be consistent with the other API implementations for now.